File: src/main/java/tgcfs/Performances/Statistics.java
package tgcfs.Performances;

import java.util.UUID;

/**
 * Created by Alessandro Zonta on 10/03/2018.
 * PhD Situational Analytics
 * <p>
 * Computational Intelligence Group
 * Computer Science Department
 * Faculty of Sciences - VU University Amsterdam
 * <p>
 * a.zonta@vu.nl
 */
public class Statistics {
    private UUID idTra;
    private double mse;

    public Statistics(UUID idTra, double mse){
        this.idTra = idTra;
        this.mse = mse;
    }

    @Override
    public String toString() {
        return "{ Trajectory: " + this.idTra.toString() + " mse: " + this.mse + " }";
    }
}

File: src/main/java/tgcfs/Performances/Statistics.kt
package tgcfs.Performances

import java.util.*

/**
 * Created by Alessandro Zonta on 21/02/2018.
 * PhD Situational Analytics
 *
 *
 * Computational Intelligence Group
 * Computer Science Department
 * Faculty of Sciences - VU University Amsterdam
 *
 *
 * a.zonta@vu.nl
 */
class Statistics(private val idTra: UUID, private val mse: Double) {

    /**
     * To string override
     * @return print the two values
     */
    override fun toString(): String {
        return "{ Trajectory: ${this.idTra} mse: " + this.mse + "}"
    }
}
